source/_docs/configuration/securing.markdown

```diff
@@ -3,14 +3,44 @@ title: "Securing"
 description: "Instructions on how to secure your Home Assistant installation."
 ---
 
-One major advantage of Home Assistant is that it's not dependent on cloud services. Even if you're only using Home Assistant on a local network, you should take steps to secure your instance.
+One major advantage of Home Assistant is that it is not dependent on cloud services.  By default, you are in control of your data. The decisions you make will affect your security and your privacy.  We take your privacy and security seriously.  Because of this, even if you are only using Home Assistant on a local network, you should take steps to secure your instance.
 
-## Checklist
+<div class='info'>
+Disclaimer: Home Assistant is not your security advisor.  Due to continually changing threat landscapes and individual configurations we recommend best practices.  The advice given here should be considered as a security baseline and should not represent your entire security profile.  
+</div>
 
-Here's the summary of what you *must* do to secure your Home Assistant system:
+### Initial Setup
+
+After installation we strongly recommend you perform the following tasks before using the system.  
+
+| Task | Justification |
+| --- | --- |
+| Setup [encrypted remote access](/docs/configuration/securing/#remote-access). | Setup HTTPS to encrypt your communication.  HTTP will allow eavesdropping on your passwords. |
+| Secure [access methods](docs/configuration/securing/#extras-for-manual-installations) | Certain installation methods have dependencies on SSH and other access methods. It is best to remove or secure these. |
+| Ensure all [Updates](https://www.home-assistant.io/common-tasks/os/#update) are applied. | Running older versions of Home Assistant can provide vulnerabilities which may be exploitable. |
```

Comment:
```suggestion
| Ensure all [Updates](https://www.home-assistant.io/common-tasks/os/#update) are applied. | Running older versions of Home Assistant can expose you to old vulnerabilities which may have been resolved in a newer release but could be exploitable until you update. |
```

Reply:
This ruins the table.  I tried to keep it as short as possible.

Reply:
OK, I think `expose you to` is better than `provide`, the connotation doesn't make sense. The rest is not strictly necessary

Reply:
Depends on your approach.  Vulnerabilities are provided to a hacker.  You provide them by failing to update.  The vulnerabilities are provided by the system to the hacker for exploitation.

Reply:
I see what you are getting at. I still think `expose you to` is better than `provide`, and all of this makes it sound like there are definitely vulnerabilities in older versions of HA. The only one I am aware of in recent memory was in January.

Reply:
There are definitely vulnerabilities.  There is no penetration testing program nor security documentation defining what is allowed and what is not.   Projects don't get this large and have 2-vulerabilities.  There is no auditing project which I'm aware of.  It's just people continuously making good decisions.   

There's nothing wrong with that.  Nothing at all.  But there's no real focus on security. As a security engineer who has looked at add-ons and a bit in supervisor, I can tell you that I'm talking no less than 10-active vulnerabilities right now and potentially thousands depending on integrations.   But it depends on the definition of user. It also depends on what people consider vulnerabilities. 

eg.  Addons start with security rating of 5.  With a 5 rating, I can wreck your entire instance, all zeros on your drive.   eg. With a 5-rating, I can make an addon which will duplicate your Home Assistant to my server.   eg. With a 6-rating,  I can inject javascript into your instance to make you perform a backup, download that backup and send it to my server.   eg. With a 6 rating I can change Host-OS configuration settings. 

I mentioned one of these and the response was something about my reputation.   It takes a shift in a project thought process to recognize security.    There are definitely vulnerabilities and they are definitely being addressed as they are recognized.